pkgs/top-level/all-packages.nix

```diff
@@ -25356,6 +25356,8 @@ in
 
   mlocate = callPackage ../tools/misc/mlocate { };
 
+  plocate = callPackage ../tools/misc/plocate { };
```

Comment:
Personally I'd prefer this either go somewhere more alphabetically appropriate or remove the newlines between `mlocate` and `plocate` like so:

```suggestion
  mlocate = callPackage ../tools/misc/mlocate { };
  plocate = callPackage ../tools/misc/plocate { };
```

Reply:
I'm aware which is why I gave the two suggestions, one not involving moving it alphabetically.
Or you can leave it as is :shrug:

Reply:
`plocate` is a different software; there is no reason to put them without whitespace.

Reply:
all-packages is not sorted at all. In the grandscheme of things this is irrelevant.